exporter/carbonexporter/README.md

```diff
@@ -6,7 +6,8 @@
 | Stability     | [beta]: metrics   |
 | Distributions | [contrib] |
 | Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aexporter%2Fcarbon%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aexporter%2Fcarbon) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aexporter%2Fcarbon%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aexporter%2Fcarbon) |
-| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@aboguszewski-sumo](https://www.github.com/aboguszewski-sumo) |
```

Comment:
Please edit the metadata files and run `make generate` instead of editing readme. Also if there are no other code owners please move these components to unmaintained

Reply:
Please also mark these components as seeking new code owners.

Here's a [recent PR](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37885) that made similar changes if it's a bit confusing. 🙂 (Refer to changes to `receiver/mysqlreceiver/metadata.yaml` to see exactly what we're requesting)